firebase_admin/_auth_utils.py

```diff
@@ -275,6 +277,131 @@ def validate_provider_ids(provider_ids, required=False):
         validate_provider_id(provider_id, True)
     return provider_ids
 
+
+def validate_mfa_config(mfa_config):
+    """Validates the specified multi factor configuration."""
+
+    mfa_config_payload = {}
+    # validate multiFactorConfig
+    if not isinstance(mfa_config, dict):
+        raise ValueError(
+            'multiFactorConfig should be of valid type MultiFactorConfig')
+
+    #validate MFA Config keys
+    valid_mfa_config_keys = set(['state', 'factorIds', 'providerConfigs'])
+    mfa_config_keys = set(mfa_config.keys())
+    for key in mfa_config_keys:
+        if key not in valid_mfa_config_keys:
+            raise ValueError('{0} is not a valid MultiFactorConfig paramter'.format(key))
+
+    # validate state in MFA config
+    if 'state' not in mfa_config:
+        raise ValueError('multiFactorConfig.state should be defined')
+    state = mfa_config['state']
+    if not isinstance(state, str) or state not in VALID_STATES:
+        raise ValueError(
+            'multiFactorConfig.state must be either "ENABLED" or "DISABLED"')
```

Comment:
nit: "must be in " + str(VALID_STATES)"?

Reply:
Same as the other comment.